pyk/src/pyk/kcfg.py

```diff
@@ -394,7 +406,29 @@ def _short_label(label):
     def _resolve_hash(self, id_like: str) -> List[str]:
         return [node_id for node_id in self._nodes if compare_short_hashes(id_like, node_id)]
 
+    def get_unique_init(self) -> Node:
+        if len(self.init) > 1:
+            raise ValueError(f'Multiple init nodes found: {list(shorten_hash(n.id) for n in self.init)}')
+        return self.init[0]
+
+    def get_unique_target(self) -> Node:
+        if len(self.target) > 1:
+            raise ValueError(f'Multiple target nodes found: {list(shorten_hash(n.id) for n in self.target)}')
+        return self.target[0]
+
+    def get_first_frontier(self) -> Node:
+        if len(self.frontier) == 0:
+            raise ValueError('No frontiers remaining!')
+        return self.frontier[0]
+
     def _resolve_or_none(self, id_like: str) -> Optional[str]:
+        if id_like == '#init':
```

Comment:
These reserved names should be forbidden in `add_alias`

Reply:
This is OK, because aliases must start with `@` and these start with `#`